core/templates/components/summary-tile/topic-summary-tile.component.spec.ts

```diff
@@ -93,4 +96,17 @@ describe('TopicSummaryTileCompoennt', () => {
   it('should get darker thumbnail background color', () => {
     expect(component.getDarkerThumbnailBgColor()).toBe('#627876');
   });
+
+  it('should get correct translations', () => {
```

Comment:
This doesn't really test that the translations are correct, AFAICT?

Reply:
Yes, it was only testing translation keys, modified the tests now